src/test/java/SetTest.java

```diff
@@ -0,0 +1,48 @@
+import java.util.HashSet;
+import java.util.Set;
+import static org.assertj.core.api.Assertions.*;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+public class SetTest {
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    @DisplayName("size 메소드를 이용해 set 의 크기를 확인하는 테스트")
+    public void setSizeTest() {
+
+        int numbersSize = numbers.size();
+
+        assertThat(numbersSize).isEqualTo(3);
```

Comment:
```suggestion
        int numbersSize = numbers.size();
        assertThat(numbersSize).isEqualTo(3);
```
코드에 흐름이 살짝 바뀔때 줄바꿈을 사용해서 환기하는 건 좋지만, 너무 줄바꿈이 잦으면 가독성을 해칠 수 있다고 생각합니다...ㅎㅎ